tests/utils/promisify.ts

```diff
@@ -18,5 +18,5 @@ test("promisify reject given fn", async (t) => {
   }
 
   const error = await t.throwsAsync(promisify(wantCb)(21));
-  t.is(error.message, "i am an error");
+  t.is(error!.message, "i am an error");
```

Comment:
could you explain this syntax? (again, I'm mostly curious)

Reply:
this is TS syntax. The error variable can be undefined (by default type is Error | undefined), in which case referring to the message property will throw error. If we don't define an if statement that checks if error is undefined, the typescript compiler will report an error. The syntax `error!.message` is such a hack that tells TS compiler that we know that error var will definitely not be undefined, and ts compiler not show error in this case.